monkey/infection_monkey/exploit/zerologon_utils/vuln_assessment.py

```diff
@@ -1,17 +1,26 @@
 import logging
-from typing import Optional, Tuple
+from typing import Tuple
 
 import nmb.NetBIOS
-from impacket.dcerpc.v5 import nrpc, rpcrt
+from impacket.dcerpc.v5 import epm, nrpc, transport
 
-from common.common_consts.timeouts import MEDIUM_REQUEST_TIMEOUT
+from common.common_consts.timeouts import LONG_REQUEST_TIMEOUT, MEDIUM_REQUEST_TIMEOUT
 from common.utils.exceptions import DomainControllerNameFetchError
 from infection_monkey.model import VictimHost
-from infection_monkey.utils.threading import interruptible_iter
 
 logger = logging.getLogger(__name__)
 
 
+def connect_to_dc(dc_ip) -> object:
```

Comment:
Let's just move all this code to the exploit file. Or at least rename it to something more appropriate.